cypress/e2e/comments.spec.ts

```diff
@@ -175,46 +228,129 @@ describe("Comments", () => {
+    describe("merged review requests", () => {
+      let reviewId: number
+      before(() => {
+        cy.intercept("GET", "**/comments").as(COMMENT_INTERCEPTOR)
+        // NOTE: Need to set permissions as order is `before` -> `beforeEach`
+        cy.setEmailSessionDefaults("Email admin")
+        cy.setupDefaultInterceptors()
+        api.closeReviewRequests()
+        cy.createEmailUser(
+          E2E_EMAIL_COLLAB.email,
+          "Email admin",
+          "Email admin",
+          E2E_EMAIL_TEST_SITE.name
+        )
+        api.editUnlinkedPage("faq.md", "some content", E2E_EMAIL_TEST_SITE.repo)
+        api
+          .createReviewRequest("test title", [E2E_EMAIL_COLLAB.email])
+          .then((id) => {
+            reviewId = id
+            cy.setEmailSessionDefaults("Email collaborator")
+            api.approveReviewRequest(reviewId).then((response) => {
+              cy.setEmailSessionDefaults("Email admin")
+              api.mergeReviewRequest(reviewId)
+            })
+          })
       })
-      it("should not be able to see comments for closed review requests", () => {
+      it("should not be able to see/create comments for merged review requests", () => {
         // Arrange
         // Act
+        cy.visit(
+          `${CMS_BASEURL}/sites/${E2E_EMAIL_TEST_SITE.repo}/review/${reviewId}`
+        )
+
         // Assert
+        checkCommentDisabled()
       })
-      it("should not be able to create comments for merged review requests", () => {
-        // Arrange
-        // Act
-        // Assert
+    })
+
+    describe("closed review requests", () => {
+      let reviewId: number
+      before(() => {
+        cy.intercept("GET", "**/comments").as(COMMENT_INTERCEPTOR)
+        // NOTE: Need to set permissions as order is `before` -> `beforeEach`
+        cy.setEmailSessionDefaults("Email admin")
+        cy.setupDefaultInterceptors()
+        api.closeReviewRequests()
+        cy.createEmailUser(
+          E2E_EMAIL_COLLAB.email,
+          "Email admin",
+          "Email admin",
+          E2E_EMAIL_TEST_SITE.name
+        )
+        api.editUnlinkedPage("faq.md", "some content", E2E_EMAIL_TEST_SITE.repo)
+        api
+          .createReviewRequest("test title", [E2E_EMAIL_COLLAB.email])
+          .then((id) => {
+            reviewId = id
+            cy.setEmailSessionDefaults("Email collaborator")
+            api.approveReviewRequest(reviewId).then((response) => {
+              cy.setEmailSessionDefaults("Email admin")
+              api.mergeReviewRequest(reviewId)
+            })
+          })
       })
-      it("should not be able to create comments for closed review requests", () => {
+      it("should not be able to see/create comments for closed review requests", () => {
         // Arrange
         // Act
+        cy.visit(
+          `${CMS_BASEURL}/sites/${E2E_EMAIL_TEST_SITE.repo}/review/${reviewId}`
+        )
         // Assert
+        checkCommentDisabled()
```

Comment:
wrong assumption - this only checks on initial load if the **button** is disabled. we need to also assert that the button is disabled _after_ the user types something into the input. (if the assertion in `checkCommentDisabled` was instead changed to be both button + input then this would be ok)

Reply:
For this test the user is unable to open the comments drawer to type stuff into the input box. Do you think I should skip this step and force the comment box to open?

Reply:
oh yea we updated the behaviour - ignore the above comment! (don't force open)